engines/director/detection_tables.h

```diff
@@ -1901,7 +1985,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	MACDEMO1("pixar", "1993 Demo", "Pixar Projector", "1ae45c23586b41997ba52e2e7c771c4c", 2324198, 310),
 
-	MACDEMO1("playroom", "Color Demo", "Color Playroom Demo", "b29b31f423a332e7f9e059187e62e1be", 1000734, 300),
+	MACDEMO1("playroom", "Color Demo", "Color Playroom Demo", "b29b31f423a332e7f9e059187e62e1be", 1000478, 300),
+	// MACDEMO1("playroom", "Demo",	   "Demo", 				  "b29b31f423a332e7f9e059187e62e1be", 1000478, 300),
```

Comment:
Is this meant to be commented? If not, can you add a second file?

Reply:
That file is the only one in the folder and is identical to the existing demo